crates/opencascade/src/primitives/wire.rs

```diff
@@ -43,6 +45,19 @@ impl Wire {
         Self::from_wire(make_wire.pin_mut().Wire())
     }
 
+    pub fn from_ordered_points(points: &[DVec3]) -> Self {
+        let mut make_wire = ffi::BRepBuilderAPI_MakeWire_ctor();
+
+        if let (Some(first), Some(last)) = (points.first(), points.last()) {
+            for window in points.windows(2).chain(once([*last, *first].as_slice())) {
```

Comment:
If we passed in just two points, would this create a "degenerate" wire which overlaps itself exactly? The code as it is here is very elegant, but I'm not sure if it handles that case of "two points" correctly.

Reply:
I tested this now a little bit and it seams to work fine with 2 different points, though it and any resulting face will result in such a "degenerate" case. Trying to extrude such a face will result an a non-triangularizable shape, so it might be easiest be to restrict this function to at least three points.

Reply:
The function now returns an error if less than three points were given.